backend/battle/migrations/0001_initial.py

```diff
@@ -1,4 +1,4 @@
-# Generated by Django 3.2 on 2021-04-28 18:39
+# Generated by Django 2.2.19 on 2021-03-22 05:52
```

Comment:
Apparently you have downgraded Django, any specific reason for that?

Reply:
Is this an older migration that you are bringing it back?

Reply:
I didn't realize that I have downgraded Django, I'm gonna fix this